cms/envs/common.py

```diff
@@ -251,6 +252,7 @@
     COMMON_ROOT / 'templates',
     COMMON_ROOT / 'djangoapps' / 'pipeline_mako' / 'templates',
     COMMON_ROOT / 'static',  # required to statically include common Underscore templates
+    OPENEDX_ROOT / 'core' / 'djangoapps' / 'cors_csrf' / 'templates',
```

Comment:
@jcdyer: Were you working on something regarding templates that may affect this?